mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -95,6 +95,7 @@ object VizReads extends BDGCommandCompanion with Logging {
   object variantsWait
   var variantsCache: Map[String, String] = Map.empty[String, String]
   var variantsRegion: ReferenceRegion = null
+  var variantsBinning: Int = -1
```

Comment:
This was to make sure the binning range got updated on my local tests. Will be removed.